linode/manager
# chore: [M3-8700] - Rename `Linode Create v2` to `Linode Create`

## Description 📝

This PR finishes the _Linode Create Reactor_ ✨  by renaming Linode `Linode Create v2` to `Linode Create` ✏️  It's been a long time coming, but we made it happen 🥲🎉

## Preview 📷

> [!note]
> There should not be any UI changes. This PR just updates file names and component names for organizational purposes 🗄️ 

## How to test 🧪

- Verify Linode Create still works as expected
- Verify unit tests pass
- Verify cypress tests pass 

## As an Author I have considered 🤔
